guides/plan-and-design.qmd

```diff
@@ -141,6 +141,10 @@ The reason that VU and its reseachers need to classify data is to understand the
 
 Data classification is about the level of sensitivity (low, medium or high) of your data assets so you can judge the risks to your research (group). This will help you when deciding what security and protection measures you need to take for handling the data or parts of the data.
 
+#### Policy Classification of Research Data
+
+The [Policy Classification of Research Data](..public/policies-regulations/Dataclassificatiebeleid-VU-NL-v1.0.pdf) addresses classification of research data in terms of availability, integrity and confidentiality, and how the classification process should be carried out. It is connected to the Research Data and Software Management Policy, because the latter states that data must be handled in a secure and reliable manner. The Research Data Classification Policy will determine what level of security measures are necessary to manage data securely.
```

Comment:
```suggestion
The [Policy Classification of Research Data](../public/policies-regulations/Dataclassificatiebeleid-VU-NL-v1.0.pdf) addresses classification of research data in terms of availability, integrity and confidentiality, and how the classification process should be carried out. It is connected to the Research Data and Software Management Policy, because the latter states that data must be handled in a secure and reliable manner. The Research Data Classification Policy will determine what level of security measures are necessary to manage data securely.
```

Missed a `/` here, which is important to ensure the path works. So `../` is always the start if you want to go up a relative folder to the current file.